tools/eslint-rules/only-ascii-characters.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Prohibit the use of non-ascii characters
+ * @author Kalon Hinds
+ */
+
+/* eslint no-control-regex:0 */
+
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const nonAsciiPattern = new RegExp('([^\x00-\x7F])', 'g');
```

Comment:
It might be clearer to use a regex literal here instead of the `RegExp` constructor. Right now, `\x00` and `\x7F` are interpreted as part of the string, so the resulting regex pattern actually contains a null character. This still works fine, but it could be confusing for debugging (e.g. if the regex is printed, it will be difficult to tell that it contains a null character).